src/test/java/org/apache/maven/plugins/javadoc/FixJavadocMojoTest.java

```diff
@@ -22,10 +22,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.StringReader;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Properties;
+import java.util.*;

```

Comment:
I'm not sure if apache maven has a style rule about this or not, but I know at Google we don't use wildcard imports.

Reply:
@elharo done.